quodlibet/quodlibet/query/_match.py

```diff
@@ -269,7 +269,10 @@ def evaluate(self, data, time, use_date):
         else:
             num = data(self._ftag, None)
         if num is not None:
-            if self._ftag in TIME_TAGS:
+            # Strip aggregate function from tag
+            func_start = self._ftag.find(":")
+            tag = self._ftag[:func_start] if func_start >= 0 else self._ftag
```

Comment:
`tag = self._ftag.split(":", 1)[0]` is handy for such things, but this is ok too.